src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/comparator/LatLong_AbsoluteDifferenceSim_Comparator.java

```diff
@@ -0,0 +1,55 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.comparator;
```

Comment:
Wirft in der Form eine NullPointerException -> raus oder verbessern

Reply:
Verbessert - an der falschen Stelle nach Null gesucht